src/containers/Home/blogposts/BlogPosts.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import BlogCard from '../../../components/Card/BlogCard/BlogCard';
+import './BlogPosts.css';
+import { Link } from 'react-router-dom';
+
+const BlogPosts = () => {
+  return (
+    <section className="flex flex-col items-center bg-secondary">
+      <h1 className=" md:text-3xl text-xl text-center font-bold text-darkgray my-10">
+        Nutrition, Health, and working out blogs
+      </h1>
+
+      <div className=" flex flex-col  items-center overflow-y-scroll  disable-scrollbar  w-screen md:h-xl h-xxl px-3">
+        <BlogCard
```

Comment:
lets make this as an array and then map over them so in the future when we get data from an API it well be easier to change